index.js

```diff
@@ -35,7 +35,7 @@ function resolveUrlLoader(content, sourceMap) {
   //  however we need to match to the sass-loader and it does not do so
   var loader     = this,
       filePath   = loader.context,
-      outputPath = path.resolve(loader.options.output.path);
+      outputPath = path.resolve(loader.options.context);
```

Comment:
* This variable should be `contextPath`. 
* Per comment below we may need to also keep `outputPath` (proposed second `try`-`catch` block).